wrfhydropy/core/job.py

```diff
@@ -23,28 +23,40 @@ def __init__(
             job_id: str,
             model_start_time: Union[str,pd.datetime] = None,
             model_end_time: Union[str,pd.datetime] = None,
-            restart_freq_hr: int = 1,
-            output_freq_hr: int = 1,
+            restart_freq_hr: int = None,
```

Comment:
what about allowing this to take multiple types, an int and a dict? If int applies to hydro and hrldas, if dict both must be specified. Might be a little clearer as to the default behavoir

Reply:
```from typing import Union```
...
```restart_freq_hr: Union[int, dict]```
... 
```
if isinstance(restart_freq_hr, dict):
    restart_freq_hr_hydro = restart_freq_hr['hydro']
    restart_freq_hr_hrldas = restart_freq_hr['hrldas']
else:
    restart_freq_hr_hydro = restart_freq_hr
    restart_freq_hr_hrldas = restart_freq_hr
```

Reply:
same for output_freq